packages/dialtone-vue3/recipes/conversation_view/editor/editor.vue

```diff
@@ -151,7 +151,7 @@
 
     <!-- Some wrapper to restrict the height and show the scrollbar -->
     <div
-      class="d-of-auto d-mx12 d-mt12 d-mb16 d-c-text"
+      class="d-of-auto d-mx16 d-mb4 d-c-text"
```

Comment:
Thanks for this fix, haven't noticed this was out of sync between vue 2 and vue 3 versions